app/views/customers/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Customer: <%= @customers.name %></h1>
+
+<h2>Items:</h2>
+<% @customers.items.each do |item| %>
+  <h3>Name: <%= item.name %></h2>
+  <h3>Price: <%= item.price %></h3>
+  <h3>Supermarket: <%= item.supermarket.name %></h3>
+<% end %>
+
+<%= form_with url: "/customers/#{@customers.id}/items" do |f| %>
```

Comment:
Don't forget to disable Turbo! You probably weren't running into problems during this IC but it might cause problems on projects or in the future so just want to share the reminder :)

Reply:
Your RESTful path is *chef's kiss*